docker-entrypoint.sh

```diff
@@ -73,6 +73,7 @@ if [[ "$TEMPLATE_CONFIG" = 'true' ]]; then
     STORAGE_TYPE_MYSQL_PASSWORD=${STORAGE_TYPE_MYSQL_PASSWORD:-"owgw"} \
     STORAGE_TYPE_MYSQL_DATABASE=${STORAGE_TYPE_MYSQL_DATABASE:-"owgw"} \
     STORAGE_TYPE_MYSQL_PORT=${STORAGE_TYPE_MYSQL_PORT:-"3306"} \
+    CERTIFICATES_ALLOWMISMATCH=${CERTIFICATES_ALLOWMISMATCH:-"true"} \
```

Comment:
I think we should set a sane and secure default by disabling this for both deployments. WDYT?

Reply:
I totally agree with you. Brought that up to Jaspreet in https://telecominfraproject.atlassian.net/browse/WIFI-11905

Reply:
Changed to false as we've discussed